hedy_translation.py

```diff
@@ -26,14 +25,15 @@ def keywords_to_dict(to_lang="nl"):
 
 
 def all_keywords_to_dict():
-    """Return a dictionary where each key is a list of the translations of that keyword. Used for testing"""    
+    """Return a dictionary where each key is a list of the translations of that keyword. Used for testing"""
     keyword_list = []
-    for lang in KEYWORD_LANGUAGES:
+    for lang in hedy_content.ALL_KEYWORD_LANGUAGES:
         commands = keywords_to_dict(lang)
         keyword_list.append(commands)
 
-    #gets the translation but defaults to the key k (in En) when it is not present
-    all_translations = {k: [d.get(k, k) for d in keyword_list] for k in keyword_list[0]}
+    # gets the translation but defaults to the key k (in En) when it is not present
+    all_translations = {k: [d.get(k, k) for d in keyword_list]
+                        for k in keyword_list[0]}
     return all_translations
```

Comment:
@boryanagoncharenko the keyword dict is filled dynamicly from now on, but that means the first item is no longer English. What does this code do exactly? Is the keyword_list[0] meant to give a default value?